snap/snapcraft.yaml

```diff
@@ -19,39 +19,101 @@ slots:
 
 apps:
   fractal:
-    extensions: [gnome-3-38]
+    extensions: [gnome]
     command: usr/bin/fractal
     common-id: org.gnome.Fractal
     desktop: usr/share/applications/org.gnome.Fractal.desktop
     plugs:
-      - audio-playback
```

Comment:
I think we need to keep this audio-playback

Reply:
I removed audio-playback and network by accident ig